pkg/controller.v1/mpi/suite_test.go

```diff
@@ -93,7 +90,7 @@ var _ = BeforeSuite(func() {
 		err = mgr.Start(testCtx)
 		Expect(err).ToNot(HaveOccurred(), "failed to run manager")
 	}()
-}, 60)
```

Comment:
@haoxins Was the timeout feature removed in ginkgo/v2?

Reply:
https://onsi.github.io/ginkgo/MIGRATING_TO_V2#removed-async-testing

Maybe we can take the [Asynchronous Assertions](https://onsi.github.io/gomega/#making-asynchronous-assertions)

Reply:
@haoxins Thanks for clarifying!

Do you @kubeflow/wg-training-leads want to fix the suite test with this PR?

Reply:
@tenzen-y Yes. Can you give LGTM?

Reply:
Yes. I meant, we can switch if there are no other concerns.  

Since `BeforeSuite` didn't use any `Done` channel, i assume that it is sill using synchronous execution and timeout is not enforced. See https://github.com/onsi/ginkgo/issues/882#issuecomment-1014693378 . 

Did you mean, we have other places in test code where we used timeouts? I see that tests are passing.

Reply:
> Since BeforeSuite didn't use any Done channel, i assume that it is sill using synchronous execution and timeout is not enforced. See https://github.com/onsi/ginkgo/issues/882#issuecomment-1014693378 .

I had missed that issue comment. Thanks, @johnugeorge.

LGTM

Reply:
@haoxins Any other changes required here to migrate to new version?

Reply:
I think no more changes required.